### PR TITLE
V1beta3

### PR DESCRIPTION
**Pull request by lovebaby979**
update broken link

### CODE REVIEW

File: keps/sig-cluster-lifecycle/clusterapi/0003-cluster-api.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ These use cases are out of scope entirely:
 
 This level of the Cluster Management API describes the global configuration of a cluster. It should be capable of representing the versioning and configuration of the entire control plane, irrespective of the representation of nodes.
 
-Given the recent efforts of SIG Cluster Lifecycle to make kubeadm the de facto standard toolkit for cloud- and vendor-agnostic cluster initialization, and because kubeadm has [an existing API](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go) to define the global configuration for a cluster, it makes sense to coalesce the global portion of the Cluster API with the API used by “kubeadm init” to configure a cluster master.
+Given the recent efforts of SIG Cluster Lifecycle to make kubeadm the de facto standard toolkit for cloud- and vendor-agnostic cluster initialization, and because kubeadm has [an existing API](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go) to define the global configuration for a cluster, it makes sense to coalesce the global portion of the Cluster API with the API used by “kubeadm init” to configure a cluster master.
```
Comment (neolit123) [inline keps/sig-cluster-lifecycle/clusterapi/0003-cluster-api.md]: as per the "recent efforts" context, this is a historic entry and i don't think we should change it.
@detiber @vincepri WDYT?

Comment (fabriziopandini) [reply keps/sig-cluster-lifecycle/clusterapi/0003-cluster-api.md:130]: IMO updating the link does not hurt anyone.
eventually, we can add a note after the link like e.g. (v1alpha3 at the time of writing this document)

**Comment by neolit123** [comment]
closing, stale PR.

**Pull request closed without merging**